notebooks/03.09-Pivot-Tables.ipynb

```diff
@@ -49,9 +49,7 @@
   {
```

Comment:
<p>Test review</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>@amit2rrr see this</p>